packages/client-glacier-node/README.md

```diff
@@ -0,0 +1,132 @@
+
+## Installing
+
+To install the this package using NPM, simply type the following into a terminal window: 
+
+```
+npm install @aws-sdk/@aws-sdk/client-glacier-node
+```
+
+## Getting Started
+
+### Import
+
+The AWS SDK is modulized by clients and commends in CommonJS modules. To send a request, you only need to import the client(`GlacierClient`) and the commands you need, for example `UploadArchiveCommand`:
+
+```javascript
+//javascript
+const { GlacierClient } = require('@aws-sdk/@aws-sdk/client-glacier-node/GlacierClient');
+const { UploadArchiveCommand } = require('@aws-sdk/@aws-sdk/client-glacier-node/UploadArchiveCommand');
+```
+
+```javascript
+//typescript
+const { GlacierClient } = import '@aws-sdk/@aws-sdk/client-glacier-node/GlacierClient';
+const { UploadArchiveCommand } = import '@aws-sdk/@aws-sdk/client-glacier-node/commands/UploadArchiveCommand';
+```
+
+### Usage
+
+To send a request, you:
+
+* Initiate client with configurations.(credentials, region)
+* Initiate command with input parameters.
+* Call `send` operation of client with command object as input.
+* If you are using customized http handler, you may call `destroy()` to close opon connections. 
+
+```javascript
+const glacier = new GlacierClient({region: 'region'});
+//clients can be shared by different commands
+const params = {
+    vaultName: /**a string value*/,
+    accountId: /**a string value*/,
+    body: /**a blob value*/
+    /**You can supply readable stream to streaming input. e.g. fs.createReadStream(file) */,
+};
+const uploadArchiveCommand = new UploadArchiveCommand(params);
+glacier.send(uploadArchiveCommand).then(data => {
+    // do something
+}).catch(error => {
+    // error handling
+})
+```
+
+Besides using promise style, there are 2 other ways to send a request:
+
+```javascript
+// async/await
+try {
+    const data = await glacier.send(uploadArchiveCommand);
+    // do something
+} catch(error) {
+    // error handling
+}
+```
+
+```javascript
+// callback
+glacier.send(uploadArchiveCommand, (err, data) => {
+    //do something
+})
+```
+ 
+Besides using `send()`, the SDK can also send requests using the simplified callback style in version 2 of the SDK.
+
+```javascript
+import * as AWS from '@aws-sdk/@aws-sdk/client-glacier-node/Glacier';
+const glacier = new AWS.Glacier({region: 'region'})
+glacier.uploadArchive(params, (err, data) => {
+    //do something
+})
+
+```
+
+For operations containing stream response like `GetJobOutput()`, you can get response stream by accessing to streaming member. e.g. data.body.pipe(/**some writable stream */).(`data` is the resolved response object)
```

Comment:
I think the formatting is a little off here.  Or, it could be a GitHub rendering issue?

Reply:
Possibly because of the `*` in the commented code. Will update

Reply:
Looks like it's just GitHub issue